### PR TITLE
Test against php 8 nightly

### PR DESCRIPTION
**Pull request by AlaaSarhan**

### CODE REVIEW

File: src/Provider/Docusign.php [hunk 1]
```diff
@@ -90,7 +90,7 @@ protected function checkResponse(ResponseInterface $response, $data)
             throw new IdentityProviderException(
                 $response->getReasonPhrase(),
                 $response->getStatusCode(),
-                $response
+                $response->getReasonPhrase()
```
Comment (AlaaSarhan) [inline src/Provider/Docusign.php:93]: Scrutinizer: This was throwing a warning due to incompatible types.

**Comment by AlaaSarhan** [comment]
Closes #10 #11

File: test/src/Provider/DocusignTest.php [hunk 2]
```diff
@@ -60,7 +60,7 @@ public function getUrlDataProvider()
         ];
     }
 
-    protected function setUp()
+    protected function setUp(): void
```
Comment (ndench) [inline test/src/Provider/DocusignTest.php:63]: Adding these return types in means that we'll have to drop support for PHP 5.6 and 7.0. But I think it's required in PHPUnit 8+ right? I think it's fine to stop supporting unsupported PHP versions though?

**Comment by AlaaSarhan** [comment]
Closing in favor of #11

**Pull request closed without merging**